### PR TITLE
Fixed Lightsleep not working properly 

### PR DESCRIPTION
**Pull request by Gladox114**
Fixed Lightsleep not working properly with really really long enough button press as it switches to the next app because doSwitch is still true form LONG_PRESS.

### CODE REVIEW

File: src/apps/main/switcher.cpp [hunk 1]
```diff
@@ -43,9 +43,10 @@ void OswAppSwitcher::loop() {
     if (hal->btnHasGoneUp(_btn)) {
         if (_doSleep) {
             _doSleep = false;
+            _doSwitch = false; // if lightsleep is enabled then doSwitch is still true and will be used the next time.
+            sleeping = true; // because in lightsleep the next _app will use the button and switch the app which isn't what we want
```
Comment (RuffaloLavoisier) [inline src/apps/main/switcher.cpp:47]: Not declared, and it should prevent initialization when it wake up.

**Review by RuffaloLavoisier — CHANGES_REQUESTED** [review]

File: src/apps/main/switcher.cpp (hunk 1 above)
Comment (Gladox114) [reply src/apps/main/switcher.cpp:47]: Whoops I forgot to add the sleeping variable initialization as I worked with two different forks and just drag and dropped my changes.

**Comment by Gladox114** [comment]
I just reopen it as I don't know how to do it properly.

**Comment by RuffaloLavoisier** [comment]
@Gladox114 
If you have any difficulties, ping, don't waste your pointless PR .

**Pull request closed without merging**